packages/proxyIdentity/contracts/IdentityManager.sol

```diff
@@ -16,38 +44,70 @@ contract IdentityManager {
     _;
   }
 
-  function identityCreated(address identity, address owner)
+  modifier isOwner() {
+    require(msg.sender == owner, "Only Identity Manager allowed");
+    _;
+  }
+  
+  modifier isOfferable() {
+    require(
+      ERC165Checker._supportsInterface(
+        msg.sender, offerableID),
+      "Only Offerable Identity allowed"
+    );
+    _;
+  }
+
+  function setLibraryAddress(address _libraryAddress) external isOwner {
+    libraryAddress = _libraryAddress;
+  }
+
+  function createIdentity(address _owner) external {
+    OfferableIdentity identity = OfferableIdentity(createClone(libraryAddress));
+    created[address(identity)] = true;
+    
+    identity.init(_owner);    
+  }
+  
+  function identityCreated(address _identity, address _owner)
     external
-    isIdentity(msg.sender, identity)
+    isIdentity(msg.sender, _identity) 
+    isOfferable
   {
-    emit IdentityCreated(identity, owner, block.timestamp);
+    if (created[msg.sender]) {
```

Comment:
👍  Nice, I think it is smart to have created be separate from verified as you have done